.github/workflows/protectAuditorsGroup.yml

```diff
@@ -0,0 +1,67 @@
+name: Protect Auditors Group
+
+on:
+  push:
+
+jobs:
+  check_membership:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Compare Group Members
+        env:
+          GH_PAT: ${{ secrets.GIT_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+
+          ##### unset the default git token (does not have sufficient rights to get team members)
+          unset GITHUB_TOKEN
+          ##### use the Personal Access Token to log into git CLI
+          echo $GH_PAT | gh auth login --with-token
+
+          # Function to get team members
+          getTeamMembers() {
+            local org=$1
+            local team=$2
+            gh api \
+              -H "Accept: application/vnd.github+json" \
+              -H "X-GitHub-Api-Version: 2022-11-28" \
+              "/orgs/$org/teams/$team/members" | jq -r '.[].login'
+          }
+
+          ORG_NAME="lifinance"
+          GROUP_A="smart-contract-admins"
+          GROUP_B="smart-contract-core"
+          GROUP_C="auditors"
```

Comment:
Can we change the variable names to fit the actual groups (i.e. SC_ADMINS or SC_CORE)? Hard to read the code later with such generic names...

Reply:
good suggestion...updated.